config/formats.js

```diff
@@ -3431,6 +3431,20 @@ exports.Formats = [
 					this.add('-anim', source, "Tail Slap", target);
 				};
 			}
+			if (move.id === 'hypervoice' && name === 'bumbadadabum') {
+				move.name = 'Open Source Software';
+				move.type = 'Electric';
+				move.basePower = 110;
+				move.accuracy = 95;
+				move.secondaries = [{chance:20, self:{boosts:{spa:-1}}, volatileStatus:'disable'}];
```

Comment:
rhythms: https://github.com/Zarel/Pokemon-Showdown/pull/1797
rhythms: does this look good for your move?
rhythms: i made the disabling chance to be 20% only
rhythms: same with the drop
%Bumbadadabum: that's fine